setup.py

```diff
@@ -25,12 +25,13 @@
     In case you want to install just one package you need to export env. variable before calling `pip`
 
      - for `pytorch-lightning` use `export PACKAGE_NAME=pytorch ; pip install .`
+     - for `lightning-lite` use `export PACKAGE_NAME=lite ; pip install .`
      - for `lightning-app` use `export PACKAGE_NAME=app ; pip install .`
 
 3. Building packages as sdist or binary wheel and installing or publish to PyPI afterwords you use command
     `python setup.py sdist` or `python setup.py bdist_wheel` accordingly.
    In case you want to build just a particular package you would use exporting env. variable as above:
-   `export PACKAGE_NAME=pytorch|app ; python setup.py sdist bdist_wheel`
+   `export PACKAGE_NAME=pytorch|app|lite ; python setup.py sdist bdist_wheel`
```

Comment:
```suggestion
   `export PACKAGE_NAME=pytorch|app|lite|lightning ; python setup.py sdist bdist_wheel`
```

Reply:
no really, `lightning` does not have any effect